documentation/developer/getting_started/04_cli.md

```diff
@@ -18,7 +18,7 @@ leo clean
 ```
 
 ```bash title="console output:"
-      Done Program workspace cleaned
+      Done Finished in 10 milliseconds
```

Comment:
This can be fixed in CLI. But for now updated as it is there

Reply:
The current output is fine.